utils/elasticache_manager.py

```
@@ -0,0 +1,329 @@
+
+def stop_cluster(name: str, region: str) -> None:
+    """Delete one ElastiCache replication group. Best-effort, logs errors."""
+    client = init_client(region)
+    logging.info(f"[elasticache_manager] Deleting replication group '{name}'")
+    try:
+        client.delete_replication_group(
+            ReplicationGroupId=name,
+            RetainPrimaryCluster=False,
+        )
+        logging.info(f"[elasticache_manager] Delete request sent for '{name}'")
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)